manifests/init.pp

```diff
@@ -58,8 +60,15 @@
     'Debian': {
       $package_name_default = 'cron'
       $service_name_default = 'cron'
+      $periodic_jobs_content_default = []
     }
     'Suse': {
+      $periodic_jobs_content_default = [
```

Comment:
This might be simpler if it was just a string with `\n` in it. Not sure that using an array buys you anything.